lang/src/org/partiql/lang/CompilerPipeline.kt

```diff
@@ -35,7 +36,12 @@ data class StepContext(
      * Includes built-in functions as well as custom functions added while the [CompilerPipeline]
      * was being built.
      */
-    val functions: @JvmSuppressWildcards Map<String, ExprFunction>
+    val functions: @JvmSuppressWildcards Map<String, ExprFunction>,
+
+    /**
+     * Returns a list of all stored procedures which are available for execution.
+     */
```

Comment:
did you want to include a note here:

> Includes built-in stored procedures as well as custom stored procedures added while the [CompilerPipeline] was being built.

?

Reply:
Added but omitted the "built-in stored procedures" part as there are no built-in PartiQL stored procedures.